PollBuddy-Server/frontend/src/pages/AccountInfo/AccountInfo.js

```diff
@@ -251,6 +244,9 @@ class AccountInfo extends Component {
   }
 
   render() {
+    if (localStorage.getItem("loggedIn") !== "true") {
```

Comment:
Wouldn't this be redundant with the `PrivateComponent` added in through `app.js`?

Reply:
Yup, just removed it.

Reply:
I think I was just using it for debugging and forgot to remove it after, thanks!